src/OrderInventory.ts

```diff
@@ -0,0 +1,103 @@
+import { genericCleanup } from "./util";
+
+/**
+ * Sort Inventories by Code
+ * TODO: only grab the inventory where the button was pressed
+ */
+export class OrderInventory {
+  private sortByCode: boolean = false;
+  private tag = "sort-by-code";
+  private cleanups: Array<() => void> = [];
+
+  cleanup() {
+    this.cleanups.forEach((f, i) => {
+      f();
+      delete this.cleanups[i];
+    });
+    genericCleanup(this.tag);
+  }
+
+  run() {
+    this.sortInventoriesByCode();
+    this.addSortByCodeButton();
+  }
+
+
+
+  private sortInventoriesByCode() {
+    if (this.sortByCode) {
+      const inventories = Array.from(
+        document.querySelectorAll("div[class^='InventoryView__grid__'")
+      );
+
+      inventories.forEach(inventory => {
+        let sorted = Array.from(inventory.children).sort(function(
+          a: HTMLElement,
+          b: HTMLElement
+        ) {
+          let codeA = a.innerText.split("\n")[0]; // ignore upper and lowercase
+          let codeB = b.innerText.split("\n")[0]; // ignore upper and lowercase
+          if (codeA < codeB) {
+            return -1;
+          }
+          if (codeA > codeB) {
+            return 1;
+          }
+          // names must be equal
+          return 0;
+        });
+        inventory.innerHTML = "";
+
+        sorted.forEach(item => {
+          inventory.appendChild(item);
+        });
+      });
+    }
+  }
+
+
+
+  private addSortByCodeButton() {
+    const buttonContainer = Array.from(
+      document.querySelectorAll(
+        "div[class^='InventorySortControls__controls___'"
+      )
+    );
+    const sortByCodeOff = () => {
```

Comment:
not sure about this: probably those should be moved into the class so they are not declared each run?